src/pretalx/api/serializers/feedback.py

```diff
@@ -0,0 +1,65 @@
+# SPDX-FileCopyrightText: 2025-present Tobias Kunze
+# SPDX-License-Identifier: AGPL-3.0-only WITH LicenseRef-Pretalx-AGPL-3.0-Terms
+
+from rest_flex_fields.serializers import FlexFieldsSerializerMixin
+from rest_framework import exceptions
+from rest_framework.serializers import SlugRelatedField
+
+from pretalx.api.serializers.mixins import PretalxSerializer
+from pretalx.api.versions import CURRENT_VERSIONS, register_serializer
+from pretalx.submission.models import Feedback, Submission
+
+
+@register_serializer(versions=CURRENT_VERSIONS)
+class FeedbackWriteSerializer(FlexFieldsSerializerMixin, PretalxSerializer):
+    submission = SlugRelatedField(
+        slug_field="code", queryset=Submission.objects.none(), source="talk"
+    )
+    speaker = SlugRelatedField(
+        slug_field="code",
+        queryset=Submission.objects.none(),
+        required=False,
+        allow_null=True,
+    )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if self.event:
+            self.fields["submission"].queryset = self.event.talks
+            self.fields["speaker"].queryset = self.event.speakers
+
+    def validate_submission(self, value):
```

Comment:
maybe also validate the speaker so only speakers of the submission can be used - or is this covered by other mechanisms?